tools/cldr-code/src/main/java/org/unicode/cldr/util/ZoneParser.java

```diff
@@ -545,7 +545,7 @@ public Comparator<String> getTZIDComparator() {
     private static List<String> errorData = Arrays.asList(new String[] {
         new Double(Double.MIN_VALUE).toString(), new Double(Double.MIN_VALUE).toString(), "" });
 
-    private Comparator<String> TZIDComparator = new Comparator<String>() {
```

Comment:
Was this change intentional?

On old Android branches we use Java 8 and this won't compile there. It is easy to revert this change, so I am just curious.

Thanks!